src/store/modules/auth.js

```diff
@@ -67,37 +67,37 @@ const getters = {
 
 const actions = {
   async setAccessToken({ commit, dispatch }, accessToken) {
-    await commit("setAccessToken", accessToken);
+    commit("setAccessToken", accessToken);
```

Comment:
why are we changing this? `fetchAndUpdateUser` is using `state.access_token` right? so, if is called before the access token is set, it will result in an error na?